docs/source/markdown/podman-container-restore.1.md

```diff
@@ -175,6 +178,15 @@ $ podman run --rm -p 2345:80 -d webserver
 # podman container restore -p 5432:8080 --import=dump.tar
 ```
 
+Start a container with name "foobar-1". Create a checkpoint image "foobar-checkpoint". Restore the container from checkpoint image with different name.
+```
+# podman run --name foobar-1 -d webserver
+# podman container checkpoint --create-image foobar-checkpoint foobar-1
```

Comment:
Here you call it `foobar-checkpoint`, but below it's `foobar-checkpoint-image`.  Does `-image` get added automatically or is it just missing in this line?

Reply:
Thanks for catching this!